tests/test_template_types.py

```diff
@@ -838,9 +839,9 @@ def test_WithSubjectTemplate_character_count(content, values, expected_count):
     ("((placeholder))  ", {"placeholder": "  "}, None, 0),
     ("  ", {}, None, 0),
     ("Content with ((placeholder))", {"placeholder": "something extra"}, "GDS", 33),
-    ("Just content", {}, "GDS", 12),
-    ("((placeholder))  ", {"placeholder": "  "}, "GDS", 5),
-    ("  ", {}, "GDS", 5),
+    ("Just content", {}, "GDS", 17),
+    ("((placeholder))  ", {"placeholder": "  "}, "GDS", 4),
+    ("  ", {}, "GDS", 4),
```

Comment:
Is this what we expect to send to the SMS providers. I was under the impression that we will send them
`GDS: ` which is 5 characters.

Reply:
we strip the space at the end if there is nothing after it, that's why 4 characters, not 5.

But actually soon we won't send them a message like this at all :). Because the function that gets SMSMessageTemplate instance in api does not provide prefix as an argument, and without prefix that message is empty.